docs/backend.md

```diff
@@ -0,0 +1,11 @@
+## Local Backend Server Setup:
+
+### Note:
+* The default is to connect to the Herokuapp at ws://scrummy-server.herokuapp.com.
```

Comment:
Can we have this read: "The deployed application connects to ws://scrummy-server.herokuapp.com."